doc/classes/Light2D.xml

```diff
@@ -417,16 +417,22 @@
 			The light texture of the Light2D is used as a mask, hiding or revealing parts of the screen underneath depending on the value of each pixel of the light (mask) texture.
 		</constant>
 		<constant name="SHADOW_FILTER_NONE" value="0">
+			Shadow filter type [code]NONE[/code]. See [method set_shadow_enabled].
```

Comment:
These descriptions just repeat the constant name. They should explain what the different modes do.

Reply:
It's just a filter for the shadow, higher = better, i don't find the right text for that.